src/ReactNativeSVG.web.ts

```diff
@@ -197,7 +197,7 @@ const prepare = <T extends BaseProps>(
     onResponderTerminationRequest?: (e: GestureResponderEvent) => boolean;
     onClick?: (e: GestureResponderEvent) => void;
     transform?: string;
-    gradientTransform?: string;
+    'gradient-transform'?: string;
     patternTransform?: string;
```

Comment:
What about `patternTransform`? Should we treat it the same way? Is there a reason why it is not automated somehow? Are there any other props that should be kebabCased?

Reply:
I think make it in a separate PR.

Reply:
No problem, but are there any other props?